ddapm_test_agent/agent.py

```diff
@@ -309,13 +336,19 @@ async def handle_session_traces(self, request: Request) -> web.Response:
         traces = await self._traces_by_session(token)
         return web.json_response(traces)
 
+    async def handle_session_tracestats(self, request: Request) -> web.Response:
+        token = request["session_token"]
+        stats = await self._tracestats_by_session(token)
+        return web.json_response(stats)
+
     async def handle_session_requests(self, request: Request) -> web.Response:
         token = request["session_token"]
         resp = []
-        for req in self._requests_by_session(token):
+        for req in reversed(self._requests_by_session(token)):
```

Comment:
why this change?

Reply:
Yeah, should have been included in #50, or earlier in `_requests_by_session`. `_requests_by_session` returns the requests in reverse order of being received.

So this is just putting them in order of being received which makes more sense to me.

I'll put up a patch following this one to reverse the order of `_requests_by_session`